Cython/Compiler/ModuleNode.py

```diff
@@ -757,9 +757,25 @@ def generate_declarations_for_modules(self, env, modules, globalstate):
             self.generate_cvariable_declarations(module, modulecode, defined_here)
             self.generate_cfunction_declarations(module, modulecode, defined_here)
 
-    @staticmethod
-    def _put_setup_code(code, name):
-        code.put(UtilityCode.load_as_string(name, "ModuleSetupCode.c")[1])
+    def _put_setup_code(self, code, name):
```

Comment:
I think #5323 has a cleaner solution to this problem, so revisit this with that in mind.